src/launchpad_mini/input.rs

```diff
@@ -28,7 +28,7 @@ fn decode_grid_button(btn: u8) -> Button {
 pub struct Input;
 
 impl crate::InputDevice for Input {
-    const MIDI_DEVICE_KEYWORD: &'static str = "Launchpad Mini";
+    const MIDI_DEVICE_KEYWORD: &'static str = "Launchpad Mini MIDI";
```

Comment:
Why this change? The MIDI device detection function (`fn guess_port`) does `.contains()` anyways, so a device called `"Launchpad Mini MIDI"` can be detected with MIDI_DEVICE_KEYWORD `"Launchpad Mini"` too. I tried to keep those keywords as generic as possible because MIDI device names are different between OSes - some OSes may append `" MIDI` to MIDI device names, but others may not